src/patternfly/components/Select/select.scss

```diff
@@ -255,11 +275,6 @@
   }
 }
 
-.pf-c-select__toggle-clear + .pf-c-select__toggle-arrow {
-  // This is adjusted so that the clear button can maintain even padding around while not creating too much space between the clear and arrow
-  --pf-c-select__toggle-arrow--MarginLeft: var(--pf-c-select__toggle-arrow--with-clear--MarginLeft);
```

Comment:
if you're removing the selector, you should remove `--pf-c-select__toggle-arrow--with-clear--MarginLeft`, too. Though I wonder if this should now become:

```
.pf-c-select__toggle-clear + .pf-c-select__toggle-button {
       --pf-c-select__toggle-arrow--MarginLeft: var(--pf-c-select__toggle-arrow--with-clear--MarginLeft); 
}
```

But update `--pf-c-select__toggle-arrow--with-clear--MarginLeft` to be 8px (`--pf-global--spacer--sm`)

Reply:
Also, if you leave this original selector, and just write a new one with what I have above, I believe this PR isn't a breaking change. You could create a follow up issue to remove `.pf-c-select__toggle-clear + .pf-c-select__toggle-arrow` with a breaking change release.

Reply:
Awesome I added it back in and will create an issue to remove it!